pytrip/ctx.py

```diff
@@ -118,7 +118,9 @@ def create_dicom(self):
             if ds.SOPInstanceUID.startswith('2.25.'):
```

Comment:
maybe this part deserves more explanation why it is needed

Reply:
I agree, but anyway DICOM related stuff needs to be refactored